### DISCUSSION TITLE
Cab, as in taxi, is short for what word? 🚕

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1199] [Galaxy Trivia] [answered]
**Bonus:** How many spark plugs does a 4-cylinder diesel engine have?

**Comment by peterbaccaro** [comment, marked as answer]
But cab is not short for taxicab; rather, it comes from **cabriolet**, which in its original use referred to a two-wheeled, one-horse carriage with a leather hood that folded down.

A 4-cylinder diesel engine has no spark plugs.  Unlike a gasoline engine (petrol engine), a diesel engine **doesn't use any spark plugs**.
